src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,36 @@
+import styles from './Modal.module.scss';
+
+export default function Modal(props: any) {
+  return (
+    <>
+      <div className={styles.modalcontainer}>
+        <div className={styles.modal}>
+          <div className={styles.modalheader}>
+            <h3>Select your home location</h3>
+            <p className={styles.close} onClick={() => props.onClose()}>
+              &times;
+            </p>
+          </div>
+          <div className={styles.modalcontent}>
+            <input type="text" />
+            <label>City to use as your default</label>
```

Comment:
a11y note: let's make label and input associated. Details: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
done)